joy: {
  # XBOX
  # safe_button: 5,
  # assign_x: 1,
  # assign_y: 3,
  # assign_z: 0,

  # ELECOM
  safe_button: 5,
  assign_x: 1,
  assign_y: 0,
  assign_z: 2,

  max_x: 4.0,
  max_y: 4.0,
  max_z: 3.0,
}

omniWheel: {
  radius: 0.05,
  wheelBase: 0.7,
}

Arduino: {
  wheel: {
    Kp: [3.0, 3.0, 3.0, 3.0],
    Ki: [0.01, 0.0, 0.01, 0.0],
    Kd: [0.02, 0.02, 0.05, 0.02]
  },
}